jinja2/testsuite/fixtures.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
```

Comment:
Couldn't we just move this file to conftest? I get it that you're trying to add some form of separation, but since conftest is empty otherwise...